docs/UserGuide.md

```diff
@@ -273,21 +351,45 @@ Examples:
 * `list` followed by `clear` clears all cases from DengueHotspotTracker.
 * `find n/Betsy` followed by `clear` deletes all cases with names that include "Betsy".
 
+<div markdown="block" class="alert alert-primary">
+
+**:exclamation: Warning:**<br>
+
+As `clear` resets all the data in `DengueHotspotTracker` you should use [`delete`](#deleting-cases-delete) for use cases of deleting specific or groups of cases.
+
+</div>
+
 <br>
 
 ### Listing all cases: `list`
 
-Displays all cases in DengueHotspotTracker.
+Use this command to reset the find filters and display all cases in DengueHotspotTracker.
 
 Format: `list`
 
+<div markdown="block" class="alert alert-primary">
+
+**:bulb: Tip:**<br>
+
+As `list` resets all active filters and sort orderings, one of the most common uses of the `list` command to reset the display lists to display all the properties and buyers in `DengueHotspotTracker`.
+Some places where you might want to make use of the `list` are when:
+* [`delete`](#deleting-cases-delete) to show all the cases as delete follows the index of the current list
+* [`overview`](#changing-the-overview-type--overview) to show to overview
+* [`find`](#finding-cases-by-prefixes-find) which filters the current list view
+
+You may wish to use `list` **before** any of these commands in order to process the *full* list of data within `DengueHotspotTracker`.
+
+You may wish to use `list` **after** any of these commands in order to return to the *full* list.
+
+</div>
+
 <br>
 
-### Finding cases by prefixes: `find` 
+### Finding cases by prefixes: `find`
```

Comment:
Hmm I'm realising that "prefix" isn't the best word here, since prefix just refers to e.g. "n/", "a/", and we're finding based on what we give _after_ the prefix. "keyword" might be a better word, but I'm not sure since using "keyword" to refer to things like dates and ages does feel a bit strange :O Do you have any suggestions?

Reply:
can be done in a future commit!

Reply:
prefer arguments instead, but me thinks removing entirely is better
```suggestion
### Finding cases: `find`
```